plugin/cmp.lua

```diff
@@ -120,10 +123,6 @@ for name in pairs(types.lsp.CompletionItemKind) do
   end
 end
 
-if vim.fn.hlexists('CmpItemMenu') ~= 1 then
-  vim.cmd [[highlight default link CmpItemMenu CmpItemMenuDefault]]
-end
-
```

Comment:
Please don't remove hlexists.
Why do you remove existing code???

Reply:
There's no need to `hlexists` if you're setting a `hi def`. `hi def` only sets the _default_ for a highlight group, so effectively, this is what was happening:

1. Check the group does not exist.
2. If it doesn't exist, set it's default. 

However, of the group did exist, it would just override the default anyway. That's why we merged #591.

By removing `hlexists`, we're getting rid of an unecessary `vim.fn` call. It was probably needed before #591 because of the bug described in that PR.

Reply:
Thank you. I've read the `highlight default` docs. And your opinion seems correct.